commit 3ff5a93f43de6c2222545cbcbf971d49a8c372ac (refs/changes/60/860/1)
Author: Patrick Mooney <pmooney@pfmooney.com>
Date:   2016-11-08T14:10:23-06:00 (2 years, 11 months ago)
    
    CAPI-528 allow credentials to differ for local/remote connections in ufds-replicator

diff --git a/sapi_manifests/ufds-replicator/template b/sapi_manifests/ufds-replicator/template
index 0008fdd..8fb309b 100644
--- a/sapi_manifests/ufds-replicator/template
+++ b/sapi_manifests/ufds-replicator/template
@@ -20,6 +20,11 @@
         ],
         "maxConnections": 1,
         "bindDN": "{{{ufds_ldap_root_dn}}}",
+        {{^ufds_remote_ldap_root_pw}}
         "bindCredentials": "{{{ufds_ldap_root_pw}}}"
+        {{/ufds_remote_ldap_root_pw}}
+        {{#ufds_remote_ldap_root_pw}}
+        "bindCredentials": "{{{ufds_remote_ldap_root_pw}}}"
+        {{/ufds_remote_ldap_root_pw}}
     }]
 }
